content/docs/reference/collections.md

```diff
@@ -1,34 +1,239 @@
+| [`defaultItem`](#defaultitem)     | An object or a function that returns an object. The object that is returned will be the data used as the default data when a new document is created.                                                                                |
+| `frontmatterFormat`               | The format used to parse the frontmatter. This can be `"yaml"` ,`"toml"`, or `"json"`. It defaults to `"yaml"`                                                                                                                       |
+| `frontmatterDelimiters`           | The Delimiters used for the frontmatter for a document. This is what Has type `string \| [string, string]` . The default is `---`. Read more about delimiters [here](https://github.com/jonschlinkert/gray-matter#optionsdelimiters) |
+| `ui.filename`                     | See [Filename customization](/docs/extending-tina/filename-customization/)                                                                                                                                                           |
+| `ui.global`                       | A boolean that if true will make this collection Global. (_optional_)                                                                                                                                                                |
+| `ui.router`                       | A function that takes in a document and returns the route for it. If nothing is returned the basic editor will be used. Read more about contextual editing [here](/docs/contextual-editing/router/#the-router-property)(_optional_)  |
+| `ui.allowedActions.create`        | If this is false, the create button will not appear in the collection list page. See [example](#example-with-allowed-actions). (_optional_)                                                                                          |
+| `ui.allowedActions.delete`        | If this is false, the create delete button will not appear in the collection list page. See [example](#example-with-allowed-actions). (_optional_)                                                                                   |
 
 > Note: Must provide only one of `fields` or `templates` but never both
 
+## Name
+
+The name of the collection that is used in the GraphQL schema. A name can only be used once and an error will occur if two or more collections have the same name. This name can can not have spaces, dashes, or special characters. It must be a valid GraphQL name. We recommend using singular naming in a collection (Ex: use post and not posts).
+
+## Path
+
+The path to a folder where the content is stored. By default, the path is relative to where the CLI is running but this can be changed by using the [`--rootPath` flag un the CLI](/docs/cli-overview). Paths do not have to be unique across collections as long as a different [`format`](#format) or [`match`](#match). The CLI will print a warning if there are overlapping documents.
```

Comment:
```suggestion
The path to a folder where the content is stored. By default, the path is relative to where the CLI is running but this can be changed by using the [`--rootPath`](/docs/cli-overview) CLI flag. Paths do not have to be unique across collections as long as a different [`format`](#format) or [`match`](#match) is specified. The CLI will print a warning if there are overlapping documents.
```